examples/demo_bgp.pp

```diff
@@ -130,46 +130,72 @@
   #---------------------------------------------------------------------------#
```

Comment:
@glennmatthews Selectors cleaned this up quite a bit.  I think this is the last change, but I don't want this merged.  Once you are ok with the changes, I'll delete this PR and make a new one to fix the history.